cwms-data-api/src/main/java/cwms/cda/data/dao/MeasurementDao.java

```diff
@@ -123,20 +117,13 @@ public void storeMeasurement(Measurement measurement, boolean failIfExists) {
      * @param officeId   - the office id
      * @param locationId - the location id of the measurement to delete
      */
-    public void deleteMeasurements(String officeId, String locationId) {
+    public void deleteMeasurements(String officeId, String locationId, Instant minDateMask, Instant maxDateMask, String unitSystem,
+                                   Number minHeight, Number maxHeight, Number minFlow, Number maxFlow, String minNum, String maxNum,
+                                   String agencies, String qualities) {
         connection(dsl, conn -> {
             setOffice(conn, officeId);
-            Number minHeight = null;
-            Number maxHeight = null;
-            Number minFlow = null;
-            Number maxFlow = null;
-            String minNum = null;
-            String maxNum = null;
-            String agencies = null;
-            String qualities = null;
-            Timestamp minTimestamp = null;
-            Timestamp maxTimestamp = null;
-            String unitSystem = null;
+            Timestamp minTimestamp = minDateMask == null ? null : Timestamp.from(maxDateMask);
+            Timestamp maxTimestamp = minDateMask == null ? null : Timestamp.from(maxDateMask);
```

Comment:
Why are you doing it this way instead of at line 88/89 where it's different?
`Timestamp maxTimestamp = OracleTypeMap.buildTimestamp(maxDateMask == null ? null : Date.from(maxDateMask));`

Reply:
because I just forgot we had something that builds that. fixed it.

Reply:
Oh man, I missed the minDateMask bug too, great catch.